node/src/neighborhood/mod.rs

```diff
@@ -2755,6 +2785,7 @@ mod tests {
             "route_query_succeeds_when_asked_for_one_hop_round_trip_route_without_consuming_wallet",
         );
         let mut subject = make_standard_subject();
+        subject.min_hops_count = Hops::OneHop;
```

Comment:
This test could be named better. Let's see:

`route_query_works_when_node_is_set_for_one_hop_and_no_consuming_wallet`

Feel free to use a better one if you come up with it.